code/modules/surgery/bodyparts/parts.dm

```diff
@@ -43,6 +44,10 @@
 	bodytype = BODYTYPE_MONKEY | BODYTYPE_ORGANIC
 	acceptable_bodytype = BODYTYPE_MONKEY
 
+/obj/item/bodypart/chest/monkey/Initialize(mapload)
+	. = ..()
+	icon_static = 'icons/mob/animal_parts.dmi'
```

Comment:
why not just set icon_static to icon

Reply:
why even set it on initialize

Reply:
Yeah, I didn't think to check if it would work without initialize. I'll run a test setting icon_static at the base obj. because I think there _may_ be some weirdness.